test/FortranArray.ll

```diff
@@ -10,7 +10,13 @@
 ; and back to .ll. This causes the LLVM IR verifier to fail as there
 ; are different rules for valid DISubRange depending on language ID.
 
+; RUN: llvm-spirv %t.bc -spirv-text --spirv-debug-info-version=nonsemantic-kernel-100
```

Comment:
Lets don't change this test in _this_ patch

Reply:
Why not? We just add the run/check lines for new behavior for DISubrange. Or do we plan to preserve _only one_ version of DISubrange translation?

Reply:
The problem here is with the Fortran language. As we discussed offline, the lower bound can start from either 0 or 1 depending on the language. So for the older spec, we won't have Fortran language, which means that after the translation, we will have OpenCL encoded in dwarf. In this test, the lower bound seems to be set as 1, causing the LLVM verifier to error out.